lms/templates/user_dropdown.html

```diff
@@ -20,7 +20,7 @@
             <img class="menu-image" src="${profile_image_url}" alt="">
             ${username}
         </a>
-        <div role="group" aria-label="User menu" class="user-menu">
+        <div role="group" aria-label="User menu">
```

Comment:
@andy-armstrong I can help out here. However, I just don't notice a bug in your sandbox. I guess I just don't know what bug you are worried about. Maybe I'll talk to you tomorrow morning to find out?